tests/system/test_root_metronome.py

```diff
@@ -127,7 +127,7 @@ def test_disable_schedule_recovery_from_master_bounce():
         client.update_schedule(job_id, 'nightly', job_schedule)
 
         # bounce metronome master
-        common.run_command_on_metronome_leader('sudo /sbin/shutdown -r now')
+        shakedown.run_command_on_leader('sudo systemctl restart dcos-mesos-master')
```

Comment:
This comment says "bounce metronome master" but the command says dcos-mesos-master. Comment needs to be updated?